designs/bee/src/band-tie.mjs

```diff
@@ -1,21 +1,22 @@
+import { pctBasedOn } from '@freesewing/core'
 import { cup } from './cup.mjs'
 
 export const bandTie = {
   name: 'bee.bandTie',
   after: cup,
   options: {
+    //Style
     bandTieWidth: {
       pct: 3,
       min: 1,
       max: 9,
-      snap: {
-        metric: [6, 13, 19, 25, 32, 38],
-        imperial: [6.35, 12.7, 19.05, 25.4, 31.75, 38.1],
-      },
-      toAbs: (pct, { measurements }) => measurements.hpsToWaistFront * pct,
+      snap: 6.35,
+      ...pctBasedOn('hpsToWaistFront'),
       menu: 'style',
     },
     bandTieLength: { pct: 35, min: 30, max: 50, menu: 'style' },
+    bandTieEnds: { dflt: 'straight', list: ['straight', 'pointed'], menu: 'style' },
+    bandTieColours: { dflt: 'one', list: ['one', 'two'], menu: 'style' },
```

Comment:
This is now the `duoColorTies` option which is a boolean rather than a 2-entry list.